ovs/extensions/services/upstart.py

```diff
@@ -91,18 +92,27 @@ def add_service(name, client, params=None, target_name=None, additional_dependen
             client.file_write('/etc/init/{0}.conf'.format(target_name), template_file)
 
     @staticmethod
-    def get_service_status(name, client):
+    def get_service_status(name, client, return_output=False):
```

Comment:
The body relies heavily on the `return_output` argument to be a boolean (what if it were `1` ?)
It might be safer to assert the the type first (something like  `assert (type(return_output) is types.BooleanType)` ).
The tests in the body become simpler:
`if return_output: ....`
Also the type of the return value isn't stable. Since you always capture the output, you might want to always return the tuple `(status, output)`. Otherwise, you force a client to change its code if it changes its mind about its interest in the output.